docs/en/documentation/known-issues-overview.md

```diff
@@ -16,7 +16,18 @@ trackSlugEN: known-issues-article
 
 [Known Issues](https://help.vtex.com/known-issues) inform users about identified problems in the VTEX platform or products, their current status, and potential solutions (workarounds) or permanent fixes. We constantly update these articles so you can anticipate potential issues and know that our team is aware of them.
 
-In this article, you’ll see the difference between Known Issues, other types of articles, and Product Improvements, as well as an overview of a Known Issue article.
+This guide covers the complete Known Issues workflow, including the differences between Known Issues and other article types, how to create them, and best practices for writing effective Known Issues documentation.
+
+## Publication workflow
+
+Each time someone from VTEX Product Support creates or updates a public Known Issue on Zendesk, an automated integration workflow is triggered:
+
+1. **Creation/Update in Zendesk**: Product Support team creates or updates a Known Issue
+2. **Automatic translation**: Content is automatically translated to Spanish and Portuguese using DeepL API
+3. **Publication**: Translated content is published or updated on Help Center
+4. **Notification**: A Slack thread is created in `#known-issues-feed` for status tracking
+
+> ℹ️ Only the VTEX Product Support team can create Known Issues. All Known Issues are created in Zendesk, not directly in the repository.
```

Comment:
Faz sentido essa info ser pública?
Acho que seria melhor colocar um link em um callout, indicando algo como "For VTEXers only. Check..." e colocar o link da doc interna onde isso é explicado